worker-core/src/main/java/com/hpe/caf/worker/core/WorkerCore.java

```diff
@@ -505,11 +508,11 @@ public void complete(final TaskInformation taskInformation, final String queue,
                     // messages.
                     workerQueue.acknowledgeTask(taskInformation);
                 } else {
-                    // **** Normal Worker ****                    
+                    // **** Normal Worker ****
                     // A worker with an input and output queue.
-                    byte[] output = codec.serialise(responseMessage);
-                    workerQueue.publish(taskInformation, output, queue, Collections.emptyMap(), 
-                                           responseMessage.getPriority() == null ? 0 : responseMessage.getPriority(), true);                    
+                    final byte[] output = convertAndSerializeMessage(responseMessage, publishTaskDataAsObject);
+                    workerQueue.publish(taskInformation, output, queue, Collections.emptyMap(),
+                                           responseMessage.getPriority() == null ? 0 : responseMessage.getPriority(), true);
```

Comment:
Again I agree with the change but it's making me spend time determining what has changed only to discover it is just whitespace.  Keep your own whitespace tidy in lines you're introducing or changing but don't go fixing it unnecessarily in lines you're not otherwise changing.  There is no issue having a separate commit for general whitespace tidy-ups.

```suggestion
                    workerQueue.publish(taskInformation, output, queue, Collections.emptyMap(), 
                                           responseMessage.getPriority() == null ? 0 : responseMessage.getPriority(), true);                    
```

Reply:
Whitespace corrections have creeped in due to auto-format. I'll see how I can correct that in IDE configuration.

Reply:
> Again I agree with the change but it's making me spend time determining what has changed only to discover it is just whitespace. Keep your own whitespace tidy in lines you're introducing or changing but don't go fixing it unnecessarily in lines you're not otherwise changing. There is no issue having a separate commit for general whitespace tidy-ups.

For now, you may see additional whitespace changes where I am changing the parameters to be on separate lines. This is being done in places where after my change the character count is spilling over 138.